src/java/org/apache/cassandra/index/sai/metrics/TableQueryMetrics.java

```diff
@@ -105,6 +120,14 @@ private void recordStringIndexCacheMetrics(QueryContext events)
             postingsDecodes.update(events.triePostingsDecodes);
         }
 
+        private void recordNumericIndexCacheMetrics(QueryContext events)
```

Comment:
nit: I can't remember why this is called `recordNumericIndexCacheMetrics` vs. `recordNumericIndexPostingsMetrics`